### PR TITLE
Sign up page

### PR DESCRIPTION
**Pull request by Temidayo24**
1. Styled the AuthenticationMainText component to be reused across pages
2. Created the create an account page, yet to be styled.
3. Created a create account css stylesheet to hold all the styles for the registration pages
4. Switched from styled components to css stylesheet in the logo component

### CODE REVIEW

File: src/components/AuthenticationMainText.css [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+.mainTextHeader {
+    font-size: 3.3em;
+    color:#00151A;
+}
+
+.mainTextBody {
+    font-size: 1.5em ;
+    color: #55666A;
+}
```
Comment (MUFTIATBAKARE) [inline src/components/AuthenticationMainText.css:14]: Please, move this file into the styles folder as it is not a component

File: src/pages/CreateAccount/SignUp.js [hunk 2]
```diff
@@ -0,0 +1,147 @@
+import React, { useState } from "react";
+import onboardingImg from "../../assets/SignUpDesign.svg";
+import "../../components/Logo";
+import Logo from "../../components/Logo";
+import "../../components/AuthenticationMainText";
```
[the remark this answers is not in the document]
Comment (MUFTIATBAKARE) [reply src/pages/CreateAccount/SignUp.js:5]: line 3 and 5 can be deleted``

File: src/pages/CreateAccount/SignUp.js [hunk 3]
```diff
@@ -0,0 +1,147 @@
+import React, { useState } from "react";
+import onboardingImg from "../../assets/SignUpDesign.svg";
+import "../../components/Logo";
```
Comment (MUFTIATBAKARE) [inline src/pages/CreateAccount/SignUp.js:3]: import what?

File: src/pages/CreateAccount/SignUp.js
Comment (MUFTIATBAKARE) [inline src/pages/CreateAccount/SignUp.js:1]: Good job!

**Review by MUFTIATBAKARE — CHANGES_REQUESTED** [review]

File: src/pages/CreateAccount/SignUp.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (Temidayo24) [reply src/pages/CreateAccount/SignUp.js:5]: Line 3 is repeated. Line 4 is the actual import.

[the remark this answers is not in the document]
Comment (Temidayo24) [reply src/pages/CreateAccount/SignUp.js:5]: Line 5 imports wasn't updated. Will make corrections.

File: src/components/AuthenticationMainText.css (hunk 1 above)
Comment (Temidayo24) [reply src/components/AuthenticationMainText.css:14]: As and the time I pushed, you had not merged. Didn't want to create another folder.

File: src/pages/CreateAccount/SignUp.js (hunk 3 above)
Comment (Temidayo24) [reply src/pages/CreateAccount/SignUp.js:3]: Will update

File: src/styles/Auth.css
Comment (Temidayo24) [inline src/styles/Auth.css:1]: Screenshot from 2023-05-24 08-04-06

**Review by MUFTIATBAKARE — APPROVED** [review]

File: src/styles/Auth.css
Comment (MUFTIATBAKARE) [reply src/styles/Auth.css:1]: I'd prefer we stick to %

**Pull request merged**